c++/src/kj/compat/http.c++

```diff
@@ -2295,12 +2298,25 @@ class WebSocketImpl final: public WebSocket {
 public:
   WebSocketImpl(kj::Own<kj::AsyncIoStream> stream,
                 kj::Maybe<EntropySource&> maskKeyGenerator,
+                kj::Maybe<CompressionParameters> compressionConfig = nullptr,
                 kj::Array<byte> buffer = kj::heapArray<byte>(4096),
                 kj::ArrayPtr<byte> leftover = nullptr,
                 kj::Maybe<kj::Promise<void>> waitBeforeSend = nullptr)
       : stream(kj::mv(stream)), maskKeyGenerator(maskKeyGenerator),
+        compressionConfig(kj::mv(compressionConfig)),
         sendingPong(kj::mv(waitBeforeSend)),
-        recvBuffer(kj::mv(buffer)), recvData(leftover) {}
+        recvBuffer(kj::mv(buffer)), recvData(leftover) {
+    #if KJ_HAS_ZLIB
```

Comment:
We normally don't intent preprocessor directives at all, though the code that they surround is indented as it would be if the preprocessor directives weren't there.

(Occasionally if preprocessor directives are deeply nested and not interleaved with regular code, I might indent them... not in places like this though.)

Reply:
https://github.com/capnproto/capnproto/pull/1505/commits/0da5d412d313a2c3ca628fbc3c6b29720f61cd16

Realized I linked the fixup to the wrong commit (previously, not this fixup ^) so I'm going to move these preprocessor directives to the correct commit next.